microservices/projects/data/projects_data.go

```diff
@@ -84,7 +84,7 @@ func (pd *ProjectData) GetProject(projectID int) (*Project, error) {
 
 func (pd *ProjectData) GetUserProjects(username string) ([]Project, error) {
 	var p []Project
-	sqlStatement := "SELECT * FROM projects WHERE project_owner_username = $1"
+	sqlStatement := "SELECT * FROM projects WHERE LOWER(project_owner_username) = $1"
```

Comment:
Are we sure we don't want case sensitive usernames?

Reply:
We were doing direct comparisons, and links force lowercase but you can put in upper case letters. So the function would fail depending on the above.

Maged said leave it like this and deal with it later

Reply:
I'm literally shaking with anger